### PR TITLE
Tsz assessment1

### PR DESCRIPTION
**Pull request by bli013**

### CODE REVIEW

**Comment by bli013** [comment]
I left #5 commented out  because #4 and #5 will run separately but not together.

File: js-intro.js [hunk 1]
```diff
@@ -10,22 +10,49 @@
...
+  else if (temp > 212){
+    return `${temp} is above boiling point`
+  }//boiling
+  else if (temp === 212){
+    return `${temp} is at boiling point`
+  }
+}
+console.log(hotOrNot(temp1))
+console.log(hotOrNot(temp2))
+console.log(hotOrNot(temp3))
```
Comment (sjproctor) [inline js-intro.js]: This function is hot for sure! Nice job.

File: js-intro.js [hunk 2]
```diff
@@ -10,22 +10,49 @@
...
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
+//creating a function using .map()
+// create newArr
+var newArr = myNumbers2.map(el => el * 5)
+console.log(newArr);
```
Comment (sjproctor) [inline js-intro.js]: Looks great - if you wrap this is a function it will become reusable.

File: js-intro.js [hunk 3]
```diff
@@ -35,27 +62,76 @@ var myNumbers2 = [8, -7, 0, 6, 2]
...
+//   // using .split to get string into individual ""'s, filter the string for just vowels
+//     return thing.split("").filter(function(el) {
+//       //brings back the vowels and string, and subtract out the vowels from joined string
+//       return vowels.indexOf(el.toLowerCase()) == -1;
+//     }).join("");
+//   }
+// }
+// // THE END OF COPIED CODE///////////////////////////
+// console.log(noVowels(notAString1));
+// console.log(noVowels(notAString2));
```
Comment (sjproctor) [inline js-intro.js]: noVowels = (thing) => {
  let vowels = ["a","e","i","o","u"]
  // using .split to get string into individual ""'s, filter the string for just vowels
  if(typeof thing === "string"){
    return thing.split("").filter(function(el) {
      //brings back the vowels and string, and subtract out the vowels from joined string
      return vowels.indexOf(el.toLowerCase()) == -1;
    }).join("")
  }
  return "Please enter a string."
}
I modified your code to use typeof for this example.

**Pull request merged**